scripts/tsrc.ts

```diff
@@ -6,7 +6,7 @@ const projectDirectory = resolve(__dirname, "..");
 
 async function main() {
 	const tsrc = new TSRC(projectDirectory, "@/tsrc", [
-		"@",
+		// "@",
```

Comment:
@hgossler I'm unsure if this is needed.

Reply:
Yes, TSRC must include the `@` dir, so that it can scan the generated route files that can possibly contain `QueryParam` type defs.